src/EnergyPlus/PlantUtilities.cc

```diff
@@ -1002,7 +1002,7 @@ void UpdateChillerComponentCondenserSide(EnergyPlusData &state,
 
     // could also check heat rate against McDeltaT from node data
 
-    if ((state.dataLoopNodes->Node(InletNodeNum).MassFlowRate == 0.0) && (ModelCondenserHeatRate > 0.0)) {
+    if ((state.dataLoopNodes->Node(InletNodeNum).MassFlowRate == 0.0) && (std::abs(ModelCondenserHeatRate) > 0.0)) {
```

Comment:
This should probably be condensed down to a single `if-else` block since there's no need to check all the conditions. 

Beyond that, I'm not quite sure I understand the logic of the changed line on 1005. `DidAnythingChange = true` if mass flow is 0 AND there is condenser heat transfer? How is that possible?

Reply:
Alright, I see the point of this function. Collapsing into a single `if-else` block is still valid, IMO, but probably not required.

Do these need the same correction?

https://github.com/NREL/EnergyPlus/blob/b819a1e19ce853c55cd3f762b27a018fa368e5fc/src/EnergyPlus/PlantUtilities.cc#L1101

https://github.com/NREL/EnergyPlus/blob/b819a1e19ce853c55cd3f762b27a018fa368e5fc/src/EnergyPlus/PlantUtilities.cc#L1177